eng/pipelines/build-unix-job.yml

```
@@ -30,8 +30,28 @@ jobs:
   steps:
     - template: checkout-unix-task.yml
 
+    - script: echo "##vso[task.setvariable variable=NUGET_PACKAGES]$(Build.SourcesDirectory)/.packages"
+      displayName: Set Nuet Packages Directory
+
+    - task: CacheBeta@1
+      displayName: .dotnet Cache
+      inputs:
+        key: 'dotnet |  $(Agent.OS) | ${{ parameters.configuration }} | $(Build.SourcesDirectory)/global.json'
```

To avoid a path-like string segment from being treated like a file path, wrap it with double quotes.

Also, not sure how configuration plays into the .dotnet cache.

```suggestion
        key: '"dotnet" |  "$(Agent.OS)" | $(Build.SourcesDirectory)/global.json'
```